pkg/arvo/sys/zuse.hoon

```diff
@@ -2235,6 +2235,107 @@
         =/  pub  (from.j qj)
         ?<  =([0 0] pub)
         pub
+      ++  schnorr
+        ~%  %schnorr  ..schnorr  ~
+        =>  |%  ++  tagged-hash
+                  |=  [tag=@ [l=@ x=@]]
+                  =+  hat=(sha-256:sha (swp 3 tag))
```

Comment:
The levels of swapping here are frustrating. `+sha-256:sha` swaps it's input and output, so this swaps thrice. It's equivalent to `(rev 3 32 (shax tag))`, but idk if it's worth changing. Just frustrating ...

Reply:
Yep. Root cause is that BIP 340 specifies things the wrong endianness so something is gonna be backwards no matter what. I opted for consistency with the `+sha` usage below.